ppr-ui/src/views/discharge/DischargeRegistration.vue

```diff
@@ -232,7 +228,7 @@ export default defineComponent({
       // do not proceed if app is not ready
       if (!val) return
       // redirect if not authenticated (safety check - should never happen) or if app is not open to user (ff)
-      if (!isAuthenticated.value || (!props.isJestRunning && !getFeatureFlag('ppr-ui-enabled'))) {
+      if (!isAuthenticated.value || !getFeatureFlag('ppr-ui-enabled')) {
```

Comment:
What's the deal with removing jestRunning check?

Reply:
Jest is gone and this is cumbersome tech-debt to be honest.
Not sure we should code in conditions to support unit testing when we can either..
a) mock functionality or components that may be problematic in tests
b) setup the test env properly so that these conditions are not an issue, ie setting the default flags in our setup for example.

Reply:
Right, no jest anymore.. I didn't like to have a check if test is running in the code anyway.
I like the option b.